pkg/scheduler/apis/config/validation/validation_pluginargs_test.go

```diff
@@ -652,17 +926,8 @@ func TestValidateNodeAffinityArgs(t *testing.T) {
 	}
 }
 
-func assertErr(t *testing.T, wantErr string, gotErr error) {
-	if wantErr == "" {
-		if gotErr != nil {
-			t.Fatalf("\nwant err to be:\n\tnil\ngot:\n\t%s", gotErr.Error())
-		}
-	} else {
-		if gotErr == nil {
-			t.Fatalf("\nwant err to be:\n\t%s\ngot:\n\tnil", wantErr)
-		}
-		if gotErr.Error() != wantErr {
-			t.Errorf("\nwant err to be:\n\t%s\ngot:\n\t%s", wantErr, gotErr.Error())
-		}
+func assertErr(t *testing.T, funcName string, wantErr, gotErr error, opts ...cmp.Option) {
```

Comment:
I am no using this func in `TestValidateNodeAffinityArgs` because it was changed at #96522

Reply:
Remove this function. Use `Diff` in the tests directly.